gkjohnson/three-mesh-bvh
# Edge projection demo

**TODO**
- [ ] ~Remove gaps when generating edges for the model (see chair back)~
- [ ] ~Reduce redundant lines~
- [x] Fix NaNs
- [x] Test with test model
  - ~Fix NaNs~
  - ~check BVH bounds~
  - ~Improve edge geometry generation (round to floor for hash?)~
  - ~Reduce memory overhead~
  - Time each step overall
  - End shapecast traversal even earlier?
- [x] Clean up demo
  - Add a "regenerate" button
  - Add a rotation configuration
  - Add model toggle
  - Display generation time
  - Simplify use
- [x] Support multiple meshes
- [x] Final debugging
